src/App.tsx

```diff
@@ -1,27 +1,48 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
+
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { Good } from './types/Good';
+import { getAll, get5First, getRedGoods } from './api/goods';
+
+import './App.scss';
+
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[] | null>(null);
```

Comment:
better to set initial state empty array if we expect array in future.
and check on goods.length
```suggestion
  const [goods, setGoods] = useState<Good[] >([]);
```